src/components/ShoppingList/ShoppingList.js

```diff
@@ -41,22 +48,48 @@ function ShoppingList({ listId }) {
     } else {
       return (
         <ul className="shopping-list__list list-reset">
-          {listItems.docs.map((doc) => (
-            <ShoppingListItem
-              key={doc.id}
-              listId={listId}
-              itemId={doc.id}
-              item={doc.data()}
-              handleCheck={handleCheck}
-            />
-          ))}
+          {listItems.docs
+            .filter((doc) => {
+              return doc.data().itemName.includes(filter.toLowerCase());
+            })
+            .map((doc) => (
+              <ShoppingListItem
+                key={doc.id}
+                listId={listId}
+                itemId={doc.id}
+                item={doc.data()}
+                handleCheck={handleCheck}
+              />
+            ))}
         </ul>
       );
     }
   };
 
   return (
     <div className="shopping-list">
+      <div className="filter">
+        <label htmlFor="filterInput" className="filter__label">
```

Comment:
I missed this when you asked me earlier, but to have it pick up our generic label styles we should put an additional class of simply `label` here too.